### DISCUSSION TITLE
Modify excluded list

### DISCUSSION THREAD

**Discussion by Dantimao** [Weatherlights/Winget-AutoUpdate-Intune#8] [Q&A] [answered]
How do we modify the excluded apps text file? It seems like its a pre built list and we are using Intune to implement Winget Auto Update Configurator

**Comment by Weatherlights** [comment]
You don't modify it. You can modify everything by using the ADMX template and set the exclude app list there.

Like in this screenshot:
![wauc applist](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/15797621/a94384e4-65ba-4fb9-9d4d-f4a35d12c773)

**Comment by flotpg** [comment]
This is unclear to me too.
Yes, I understand you use the GPO / Intune ADMX to configure exclude list or white list (if you set to whitelist mode).

If I run it in exclude list mode ("Use White List instead of Black List" = not configured), it still uses the "C:\ProgramData\Winget-AutoUpdate\**excluded_apps.txt**"
```
Google.Chrome
Microsoft.Edge
Microsoft.EdgeWebView2Runtime
Microsoft.Office
Microsoft.OneDrive
Microsoft.Teams
Mozilla.Firefox
Mozilla.Firefox.DeveloperEdition
Mozilla.Firefox.ESR
TeamViewer.TeamViewer
TeamViewer.TeamViewer.Host
```

So apps like Teams, Office, etc won't get updated:
`08:56:18 - Microsoft Teams : Skipped upgrade because it is in the excluded app list`

Why are the apps above on the exclude list and how do I remove them?

Many thanks and best regards, Flo.

**Comment by Weatherlights** [comment]
Hey

I analyzed what is happening and as a quick relief approach I recommend you to just create a blacklist with a placeholder in it, so something like doesnotexist.doesnotexist.

WAUs default behaviour is to use a default blacklist (the one you mention here) when  no blacklist is provided. These apps usually work better when you do not update them with winget but with their built in automatic update mechanisms.

By providing a blacklist with a dummy value the list is overwritten.

I will investigate on how much sense it makes to keep this list.

Hope this helps.

Best
Hauke

**Reply by flotpg** [reply]
Hey Hauke,

I was thinking about this ;) and will try that soon. Should that also create an exclude...txt file like the include mode does?

Thanks and best regards, Flo.

**Reply by Weatherlights** [reply]
Exactly: It will create an **excluded_apps.txt** file under the **"C:\ProgramData\Winget-AutoUpdate-Configurator"** folder (This folder is used for staging lists and mods to WAU). Form that location WAU will synchronize the file to it's own folder.

**Reply by flotpg** [reply]
Awesome, that worked great for me!
Thank you very much!

**Reply by STRIF-007** [reply]
how do i configure that per policies/ Intune?

**Reply by IESFelix** [reply]
> how do i configure that per policies/ Intune?

Have you figured that out?

**Reply by Weatherlights** [reply]
> > how do i configure that per policies/ Intune?
> 
> Have you figured that out?

You import the ADMX file and then create an ADMX backed policy in Intune. From there you configure the Application List (As seen in the screenshot above).

**Comment by 6wendy9** [comment]
I am a little confused on the resolution above. How do I create a custom blacklist in Intune. Do I use the Application List template or Use White List instead of Black List

**Reply by ak47uk** [reply]
Using the ADMX template in Intune, enable the option to use a blacklist instead of whitelist, then enable the application list setting and enter in all the IDs you want to blacklist. My blacklist contains apps that reliably auto-update themselves such as web browsers and Teams.

**Reply by 3m1l1ano** [reply]
Hi, 
i am also having some issues with Winget-AutoUpdate failing updating some Windows "internal" Apps.
I was thinking to exclude them but I dind't really understand how to use the Black List on White List honestly.
In fact the option isn't there. I have the WinGet-AutoUpdate-Configurator.admx 1.04, which should be the last one?
Would be great if actually can update them but that looks harder.
Thanks

**Reply by incedIT** [reply]
> Hi, i am also having some issues with Winget-AutoUpdate failing updating some Windows "internal" Apps. I was thinking to exclude them but I dind't really understand how to use the Black List on White List honestly. In fact the option isn't there. I have the WinGet-AutoUpdate-Configurator.admx 1.04, which should be the last one? Would be great if actually can update them but that looks harder. Thanks

If using Intune, have you created a new config profile, selected Templates, Imported Administrative templates (Preview)? If you are using the policies locally, then I think the path should be: Computer Configuration\Winget-AutoUpdate-Configurator

**Reply by 3m1l1ano** [reply]
Hi, thanks for the reply but still I can't see it.. see screenshot.
<img width="252" alt="Screenshot_1" src="https://github.com/user-attachments/assets/f8f3ddcc-f5bb-4308-a563-63753417e7cd">

**Reply by incedIT** [reply, marked as answer]
You now need to Enable the "Application List" and add the apps, because you have enabled "Use White List instead of Black List", the Application List will define which apps will update. I prefer to set "Use White List instead of Black List" to disabled, enable "Application List", then any apps I put in there will be ignored.

**Comment by 3m1l1ano** [comment]
Thanks!! I will like this.

<img width="293" alt="Screenshot_1" src="https://github.com/user-attachments/assets/d3680275-e8f2-4cd5-99f0-0396f6d473fb">

**Comment by incedIT** [comment]
I am currently troubleshooting a device where my blacklist has been ignored. It is possible that "Use White List instead of Black List" should be set to not configured to use a blacklist, enabled to use a whitelist. I can't find documentation, the issue is that even though Microsoft.WindowsTerminal and Windows.DevHome are on my list, they have tried to update (and failed).

**Reply by 3m1l1ano** [reply]
Windows.DevHome we updated it manually from the store.. then it will stop, sad that the auto update fails.. If we were talking about an app lile Chrome or Adobe I would be more concern about it but Windows internal apps we try to do not care much.
Best

**Reply by 3m1l1ano** [reply]
it's just annoying to get the pop up every time that it tries to update and fail.

**Reply by incedIT** [reply]
> it's just annoying to get the pop up every time that it tries to update and fail.

Yes, that is why I have it on the blacklist so it is ignored, but I had a report of the fail pop-up today along with Terminal which is also on my blacklist, so figuring out why it isn't working as expected.

**Reply by Weatherlights** [reply]
Have you checked the registry if the value was set accordingly? Could you please provide the log files?

You can find them in c:\windows\temp. You can capture the logfiles using the Intune collect logs feature.

**Reply by incedIT** [reply]
> Have you checked the registry if the value was set accordingly? Could you please provide the log files?
> 
> You can find them in c:\windows\temp. You can capture the logfiles using the Intune collect logs feature.

Will do, I am waiting to collect the logs at the moment. Do you know if setting "Use White List instead of Black List" to Not Configured/Disabled results in the Application List being used as a Blacklist? Or should it be Not Configured?

**Reply by Weatherlights** [reply]
> > Have you checked the registry if the value was set accordingly? Could you please provide the log files?
> > You can find them in c:\windows\temp. You can capture the logfiles using the Intune collect logs feature.
> 
> Will do, I am waiting to collect the logs at the moment. Do you know if setting "Use White List instead of Black List" to Not Configured/Disabled results in the Application List being used as a Blacklist? Or should it be Not Configured?

Disabled and not configured should result in Blacklist. Both values should result in the same: The -UseWhiteList parameter will not be set. The WAU setup will then set the configured list as blacklist.

**Reply by incedIT** [reply]
Here is an extract from the log to show today, it shows the correct path to the excluded_apps.txt list and on that list is "Microsoft.WindowsTerminal", yet I had "Windows Terminal could not be updated" message at 8:10 this morning:
`<![LOG[Configuration received from HKLM:\SOFTWARE\Policies\weatherlights.com\Winget-AutoUpdate]LOG]!><time="08:08:28.752-60" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Parsed list to C:\ProgramData\Winget-AutoUpdate-Configurator\excluded_apps.txt.]LOG]!><time="08:08:28.89860" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Started mod creation.]LOG]!><time="08:08:28.93660" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[No Mods location detected.]LOG]!><time="08:08:28.95160" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Finished mod creation.]LOG]!><time="08:08:29.07560" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Commandline arguments -silent -DoNotUpdate -DisableWAUAutoUpdate -NoClean -NotificationLevel Full -ModsPath "C:\ProgramData\Winget-AutoUpdate-Configurator\mods" -UpdatesInterval Weekly -UpdatesAtLogon -StartMenuShortcut -InstallUserContext -ListPath "C:\ProgramData\Winget-AutoUpdate-Configurator\" generated.]LOG]!><time="08:08:29.14460" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Previous commandline arguments -silent -DoNotUpdate -DisableWAUAutoUpdate -NoClean -NotificationLevel Full -ModsPath "C:\ProgramData\Winget-AutoUpdate-Configurator\mods" -UpdatesInterval Weekly -UpdatesAtLogon -StartMenuShortcut -InstallUserContext -ListPath "C:\ProgramData\Winget-AutoUpdate-Configurator\".]LOG]!><time="08:08:29.22360" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Skipped updating WAU.]LOG]!><time="08:08:29.25960" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Stored commandline arguments.]LOG]!><time="08:08:29.28560" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">
<![LOG[Created copy of logfiles for intune diagnostics.]LOG]!><time="08:08:29.35460" date="10-16-2024" component="WinGet-AutoUpdate-Configurator" context="NT AUTHORITY\SYSTEM" type="1" thread="19696" file="">`

Do you know where in the reg the keys would be?

**Reply by incedIT** [reply]
Is it possible that the v2.0.0 of WAU has caused this?
https://github.com/Romanitho/Winget-AutoUpdate/releases/tag/v2.0.0
> Registry key for configurations moved to HKLM\SOFTWARE\Romanitho\Winget-AutoUpdate, which can affect administrators.

My HKLM\SOFTWARE\Romanitho\Winget-AutoUpdate\WAU_ListPath key is blank.

**Comment by 3m1l1ano** [comment]
Is it much better the v2.0.0? or i can stay on the 1.05 easy :)

**Reply by Weatherlights** [reply]
The 2.0.0 version is from WAU. While the 1.05 is WAUaaS. Both are separate projects. The idea behind WAUaaS is to enhance the integration of WAU into Intune by using the Microsoft Store as primary method of distribution and using Intune configuration profiles. Many improvements are now also available in WAU itself. I guess the most distinct one in this project are still:
- Available through the Microsoft store and whitelisted by Defender Smartscreen
- Easier way of using modifications by avoiding the use of additional ressources outside intune
- Signed code to make easier use of Endpoint Privilege Management or Application Control

**Reply by 3m1l1ano** [reply]
ok! thank you :)

**Reply by fabssnn** [reply]
Hi all

I use the 1.05 which I roll out in intune via webstore, additionally I use the admx to distribute the gpos.

A colleague has now informed me that the blacklist function has recently stopped working.

Then I realised that for some reason the Winget-Autoupdater from Project Romanitho is also being installed. 
This seems to be happening on all devices. This was not done manually. 

![IMG_0789](https://github.com/user-attachments/assets/8c2eadb1-6ec5-4d56-9426-f4a8265d4c58)

**Reply by incedIT** [reply]
I think this is related to my post here: https://github.com/Weatherlights/Winget-AutoUpdate-Intune/discussions/8#discussioncomment-10956532
Looks like the source project has changed regkeys so this fork will need updating.

**Reply by fabssnn** [reply]
> I think this is related to my post here: [#8 (reply in thread)](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/discussions/8#discussioncomment-10956532) Looks like the source project has changed regkeys so this fork will need updating.

What can I do in this case? 
I have several devices here that are affected :)

The only workaround that works coulkd be
- I uninstall aaS and WAU on all devices
- Roll out aaS 1.05 again, so that WAU 1.20 is installed
- In the exclusion I add “Romanitho.Winget-AutoUpdate” to prevent that WAU will update to 2.0

Or is there another possibility?

**Reply by incedIT** [reply]
> > I think this is related to my post here: [#8 (reply in thread)](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/discussions/8#discussioncomment-10956532) Looks like the source project has changed regkeys so this fork will need updating.
> 
> What can I do in this case? I have several devices here that are affected :)
> 
> The only workaround that works coulkd be
> 
> * I uninstall aaS and WAU on all devices
> * Roll out aaS 1.05 again, so that WAU 1.20 is installed
> * In the exclusion I add “Romanitho.Winget-AutoUpdate” to prevent that WAU will update to 2.0
> 
> Or is there another possibility?

I think the easier fix would be to deploy the updated registry key using a script, I have not had time to try it yet. I am not sure how this fork decides what WAU version to install, it may install the latest version.

**Reply by Weatherlights** [reply]
> > > I think this is related to my post here: [#8 (reply in thread)](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/discussions/8#discussioncomment-10956532) Looks like the source project has changed regkeys so this fork will need updating.
> > 
> > 
> > What can I do in this case? I have several devices here that are affected :)
> > The only workaround that works coulkd be
> > 
> > * I uninstall aaS and WAU on all devices
> > * Roll out aaS 1.05 again, so that WAU 1.20 is installed
> > * In the exclusion I add “Romanitho.Winget-AutoUpdate” to prevent that WAU will update to 2.0
> > 
> > Or is there another possibility?
> 
> I think the easier fix would be to deploy the updated registry key using a script, I have not had time to try it yet. I am not sure how this fork decides what WAU version to install, it may install the latest version.

Nope... However I have implemented a fix that prevents this from happening. Everything is ready and it is currently undergoing the required certifications and then it will rollout. The fix will prevent WAU from updating using Winget which should clearly not happen since it would destroy the code integrity. For now this fork stays on 1.x but see how version 2 can be implemented :).

Sorry for the problems guys... I am trying to resolve this as fast as possible.

**Reply by fabssnn** [reply]
@Weatherlights 
Many thanks for your update.
Would there also be a way to fix the devices which are already affected by to enrollment of WAU 2.0?

**Reply by Weatherlights** [reply]
> @Weatherlights Many thanks for your update. Would there also be a way to fix the devices which are already affected by to enrollment of WAU 2.0?

Yes the new version will automatically remove WAU 2.0 as part of the update process and install version 1.x again.

Here is a link to the current internal Beta version: https://wuacreleases.z6.web.core.windows.net/1.06.0033.msi However it still lacks the Defender Smartscreen approval and is waiting for the store submission. But for those who need quick relief or need to validate things :).

**Reply by fabssnn** [reply]
Hi @Weatherlights 
there is nothing new regarding Microsoft Store Application yet, right?
Thanks in advance

**Reply by Weatherlights** [reply]
> Hi @Weatherlights there is nothing new regarding Microsoft Store Application yet, right? Thanks in advance

The Defender Smartscreen Team has whitelisted the release now. Now the submission to the store will begin. It will pass likely by the end of the week when nothing comes in the way.

**Reply by fabssnn** [reply]
@Weatherlights 
Thanks for your reply.

**Comment by fabssnn** [comment]
I have just tested 1.06. I've installed 1.06 manually.
I noticed that the excluded_apps.txt that I configured via Intune has been extended with the default entries that the WAU app provides.
Google.Chrome
Microsoft.Edge
Microsoft.EdgeWebView2Runtime
Microsoft.Office
Microsoft.OneDrive
Microsoft.Teams
Microsoft.Teams.Classic
Mozilla.Firefox*
TeamViewer.TeamViewer
Microsoft.RemoteDesktopClient

**Reply by dxenandx** [reply]
I have the same problem in our Environment, since the Update the Winget Excluded List is configured back to default. Which updates Apps which should not be updated.

**Reply by NickBBD** [reply]
I get the same, so to clarify, if I want to now update Teamviewer and Microsoft Office, how do i get them out of the excluded list?

**Reply by Weatherlights** [reply]
Fix is underway :).

**Comment by spitty78** [comment]
Any update to the blacklist reverting back to default or ignoring the configured blacklist from the admx in intune?

Also, what should be installed?  Both the Winget-AutoUpdate - Romanitho AND Winget-Autoupdate-aas - Hauke Goetze?
or did something get messed up in my install with the major version updates?

**Reply by Weatherlights** [reply]
Winget-Autoupdate-aas - Hauke Goetze will install Winget-AutoUpdate - Romanitho. So everything fine with that. The list stuff now has a fix and is currently in the process of being published.

**Reply by fedthans** [reply]
as i see is the WAU old version, and try manual to install WAU 2.0 but it not effect the Winget-Autoupdate-aas it still the 1.2 v !
Should this Winget-Autoupdate-aas not get updated by MS Strore? and when is the updated add to MS Store?

Have also issue regarding control the Excludelist, and that is a problem because there apps i will update, the Romanitho.Winget-AutoUpdate is in the Exclude list! why is this in the list OOTB :-)

**Reply by Weatherlights** [reply]
> as i see is the WAU old version, and try manual to install WAU 2.0 but it not effect the Winget-Autoupdate-aas it still the 1.2 v ! Should this Winget-Autoupdate-aas not get updated by MS Strore? and when is the updated add to MS Store?
> 
> Have also issue regarding control the Excludelist, and that is a problem because there apps i will update, the Romanitho.Winget-AutoUpdate is in the Exclude list! why is this in the list OOTB :-)

Simply do not service WAU yourself :). It will be maintained by WAUaaS. The 1.x version of WAU shipped is fully supported by WAUaaS. With the next release of WAUaaS (likely next week) WAU 2.8.0 is included.

**Comment by fabssnn** [comment]
Hello all,
similar problem for me. It seems that the application list admx entries are ignored and/or extended from the default excluded_apps.txt

Sample:
Following application list entries was made in intune.
<img width="440" alt="image" src="https://github.com/user-attachments/assets/6b76aada-2394-4c1c-a7cb-648f782d1207">

In Folder C:\Program Files (x86)\Winget-Autoupdate-aaS\Sources\WAU
the excluded_apps.txt has following entries:
```
Google.Chrome
Microsoft.Edge
Microsoft.EdgeWebView2Runtime
Microsoft.Office
Microsoft.OneDrive
Microsoft.Teams
Microsoft.Teams.Classic
Mozilla.Firefox*
TeamViewer.TeamViewer*
Microsoft.RemoteDesktopClient
```

In Folder C:\ProgramData\Winget-AutoUpdate-Configurator
The excluded_apps.txt has following entries:
```
Microsoft.Teams.Classic
Microsoft.PowerToys
KDE.KDiff3
CoreyButler.NVMforWindows
Microsoft.VisualStudio.2022.BuildTools
JRSoftware.InnoSetup
ARP\Machine\X86\{73A0578F-96C5-40C0-B1E7-9114520A2C96}
Microsoft.Office
Microsoft.Edge
Google.Chrome
Microsoft.VCLibs.Desktop.14
Microsoft.VCLibs.140.00.UWPDesktop
Microsoft.SQLServerManagementStudio
Microsoft.WindowsAppRuntime.1.5
Microsoft.VCRedist.2015+.x86
Microsoft.Teams
Microsoft.OneDrive
Fortinet.FortiClientVPN
OpenJS.NodeJS.LTS
Microsoft.EdgeWebView2Runtime
Mozilla.Firefox*
TeamViewer.TeamViewer*
Microsoft.RemoteDesktopClient
Romanitho.Winget-AutoUpdate
```

In Folder C:\ProgramData\Winget-AutoUpdate
The excluded_apps.txt has following entries:
```
Microsoft.Teams.Classic
Microsoft.PowerToys
KDE.KDiff3
CoreyButler.NVMforWindows
Microsoft.VisualStudio.2022.BuildTools
JRSoftware.InnoSetup
ARP\Machine\X86\{73A0578F-96C5-40C0-B1E7-9114520A2C96}
Microsoft.Office
Microsoft.Edge
Google.Chrome
Microsoft.VCLibs.Desktop.14
Microsoft.VCLibs.140.00.UWPDesktop
Microsoft.SQLServerManagementStudio
Microsoft.WindowsAppRuntime.1.5
Microsoft.VCRedist.2015+.x86
Microsoft.Teams
Microsoft.OneDrive
Fortinet.FortiClientVPN
OpenJS.NodeJS.LTS
Microsoft.EdgeWebView2Runtime
Mozilla.Firefox*
TeamViewer.TeamViewer*
Microsoft.RemoteDesktopClient
Romanitho.Winget-AutoUpdate

```

```
#################################################################
#     11/11/2024 - CHECK FOR APP UPDATES (System context - Connected user)
#################################################################
10:02:22 - Notification Level: None. Notification Language: English
10:02:22 - Checking internet connection...
10:02:23 - Connected !
10:02:23 - Winget Version: v1.9.25180
10:02:23 - WAU current version: 1.21.13
10:02:23 - WAU AutoUpdate is Disabled.
10:02:23 - WAU uses External Lists from: C:\ProgramData\Winget-AutoUpdate-Configurator
10:02:23 - Newer List downloaded/copied to local path: C:\ProgramData\Winget-AutoUpdate
10:02:24 - WAU uses External Mods from: C:\ProgramData\Winget-AutoUpdate-Configurator\mods
10:02:24 - Mods are up to date.
10:02:24 - WAU uses Black List config
10:02:24 - -> Successsfully loaded local excluded apps list.
10:02:24 - Checking application updates on Winget Repository...
-> Available update : Winget-AutoUpdate (WAU). Current version : 1.21.13. Available version : 2.1.0.
-> Available update : Microsoft Edge WebView2 Runtime. Current version : 130.0.2849.56. Available version : 130.0.2849.80.
-> Available update : TeamViewer. Current version : 15.58.5. Available version : 15.59.3.
**-> Available update : Node.js. Current version : 14.21.3. Available version : 22.11.0.**
-> Available update : Microsoft .NET SDK 8.0.110 (x64). Current version : 8.0.110. Available version : 8.0.403.
10:02:29 - Winget-AutoUpdate (WAU) : Skipped upgrade because it is in the excluded app list
10:02:29 - Microsoft Edge WebView2 Runtime : Skipped upgrade because it is in the excluded app list
10:02:29 - TeamViewer : Skipped upgrade because it is *wildcard* in the excluded app list
**10:02:31 - Updating Node.js from 14.21.3 to 22.11.0...**
10:02:31 - ##########   WINGET UPGRADE PROCESS STARTS FOR APPLICATION ID 'OpenJS.NodeJS.LTS'   ##########
**10:02:31 - -> Running: Winget upgrade --id OpenJS.NodeJS.LTS -e --accept-package-agreements --accept-source-agreements -s winget -h
```

**Reply by 3m1l1ano** [reply]
To me the excluded apps.txt that is actually working are the ones in; C:\ProgramData\Winget-AutoUpdate, C:\ProgramData\Winget-AutoUpdate-Configurator. Not sure tho why Edge, firefox etc are there when i didn't specify to exclude them.. how can I remove those from the list for everyone?
Thanks

**Reply by Weatherlights** [reply]
Fix is underway. I removed the entries from the list and prepared an update which is now going through the publishing process.

**Comment by Weatherlights** [comment]
Hey


I am currently on holiday but will look into it next week.


Best regards

--
Hauke Götze
E-Mail: ***@***.***
________________________________
Von: fabssnn ***@***.***>
Gesendet: Mittwoch, 20. November 2024 08:58
An: Weatherlights/Winget-AutoUpdate-Intune
Cc: Hauke Götze; Mention
Betreff: Re: [Weatherlights/Winget-AutoUpdate-Intune] Modify excluded list (Discussion #8)


Hello all,
similar problem for me.
It seems that the application list admx entries are ignored.

**Reply by fabssnn** [reply]
> Hey I am currently on holiday but will look into it next week. Best regards
> […](#)
> -- Hauke Götze E-Mail: ***@***.***
> ________________________________ Von: fabssnn ***@***.***> Gesendet: Mittwoch, 20. November 2024 08:58 An: Weatherlights/Winget-AutoUpdate-Intune Cc: Hauke Götze; Mention Betreff: Re: [Weatherlights/Winget-AutoUpdate-Intune] Modify excluded list (Discussion #8) Hello all, similar problem for me. It seems that the application list admx entries are ignored. — Reply to this email directly, view it on GitHub<[#8 (comment)](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/discussions/8#discussioncomment-11319118)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADYQ25OR6T2HE563IUP2NJD2BQ6JXAVCNFSM6AAAAABMJII3C2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZRHEYTCOA>. You are receiving this because you were mentioned.Message ID: ***@***.***>

I wanted to ask whether you have already had a chance to look at this problem?

**Reply by STRIF-007** [reply]
I would also what to know...

**Reply by Weatherlights** [reply]
Yea I needed a few days to recover from holiday ;). The certification with Microsoft is always a pain in the... you know what. Anyhow I fixed the list now and I am working on publishing it... lets hope the smartscreen team is in good mood :).

**Comment by Weatherlights** [comment]
Hello they apps are coming from a recommended list of the upstream project. I will exclude those entries with the next update.

________________________________
Von: STRIF-007 ***@***.***>
Gesendet: Donnerstag, 12. Dezember 2024 13:54
An: Weatherlights/Winget-AutoUpdate-Intune ***@***.***>
Cc: Hauke Götze ***@***.***>; Mention ***@***.***>
Betreff: Re: [Weatherlights/Winget-AutoUpdate-Intune] Modify excluded list (Discussion #8)


how do i configure that per policies/ Intune?

**Reply by 3m1l1ano** [reply]
thanks

**Reply by Weatherlights** [reply]
Ok fixed the list and build a new release. Is now in testing and certification with Microsoft.

**Reply by spitty78** [reply]
Has the new app been approved by Microsoft?  My implementation seems "stuck" so I am wondering if a new version will kick it in the pants to get it going again?

**Reply by WalterSobchak9515** [reply]
> Has the new app been approved by Microsoft? 

I am wondering the same, been a long time since the release in October.

**Comment by Weatherlights** [comment]
Not yet. One submission they did not even notice stuck in progress for ages :(. Made another submission yesterday.
________________________________

**Reply by spitty78** [reply]
Ugh... Microsoft! Thank you for the update.

**Reply by WalterSobchak9515** [reply]
That sucks, thank you!

**Reply by fedthans** [reply]
Is the new version of WAU 2.0 also getting opdated in the Updated?

**Reply by Weatherlights** [reply]
Not yet. I needed a little time to think about the MSI approach and what that implies to this release. However I sorted stuff out and I will hopefully make it today or tomorrow to include version 2.20 and then build the next release.

**Comment by new-it** [comment]
@Weatherlights we are using your tool since this week and we are also struggeling with the $forbiddenApps inside Refresh-WingetConfiguration.ps1

Is there any reason why this exists at all? In the admx files in intune we set it to blacklist and excluded already all apps we don't want to update.

We are using 1.06.0034 from the MS store.

**Reply by den111111111** [reply]
Same here. The default entries in the excluded_apps.txt are still there.

**Reply by Weatherlights** [reply]
Hey. Yes the "Forbidden apps" list in version 1.06.0034 has some entries that are removed in version 1.06.0036. 1.06.0036 ist currently in certification with Microsoft and will (if the Smartscreen detection with Microsoft is already up-to-date) be available in the next 1-2 days. So hold tight relief is on its way :).

**Reply by Champ895** [reply]
Do you also have a fix to make the auto-updater work in Azure Virtual Desktop (AVD)? It seems like the auto-updater only works on physical endpoints but not in a virtual machine.

**Reply by WalterSobchak9515** [reply]
Any idea when this will be passed through defender smart screen?  It's been months.

**Reply by Weatherlights** [reply]
It has been certified today and is now available... Sry... I really do not make this stuff up but Microsofts Smartscreen process is a pain in the... First it takes 1-3 weeks to have the smartscreen whitelist (usually it takes 2-3 turns) and afterwards the Microsoft Store team needs 2-3 submissions (each takes 2-3 days) to recognize this... But finally they it is through...

**Reply by WalterSobchak9515** [reply]
> It has been certified today and is now available... Sry... I really do not make this stuff up but Microsofts Smartscreen process is a pain in the... First it takes 1-3 weeks to have the smartscreen whitelist (usually it takes 2-3 turns) and afterwards the Microsoft Store team needs 2-3 submissions (each takes 2-3 days) to recognize this... But finally they it is through...

Ok, I just want to verify that the MSI file located here is approved via smart screen:  https://github.com/Weatherlights/Winget-AutoUpdate-Intune/releases/tag/1.06.0034

**Reply by Weatherlights** [reply]
Yes. I just published the 1.06.0036 release which is also smartscreen approved.

**Reply by WalterSobchak9515** [reply]
> Yes. I just published the 1.06.0036 release which is also smartscreen approved.

thank you!!!

**Reply by Champ895** [reply]
How are we updating this? I used the windows store app to deploy originally with the admx you had.

**Reply by WalterSobchak9515** [reply]
> How are we updating this? I used the windows store app to deploy originally with the admx you had.

Supposedly the store app auto updates, I have no idea.  I deploy the msi via intune.

**Reply by Champ895** [reply]
Nice - I just verified and it did in fact update and the excluded/included list now works. Thanks!

**Reply by Weatherlights** [reply]
> > How are we updating this? I used the windows store app to deploy originally with the admx you had.
> 
> Supposedly the store app auto updates, I have no idea. I deploy the msi via intune.

Yes exactly. The Store will automatically update the app. With Intune you just use a LoB App Type and replace the MSI.

**Reply by Champ895** [reply]
Cool - and a bit off topic, but do you know how we can get this app specifically working to update within Azure Virtual Desktop? We have tried previously and it doesn't work - not applicable specifically for multi-session

**Reply by Weatherlights** [reply]
> Cool - and a bit off topic, but do you know how we can get this app specifically working to update within Azure Virtual Desktop? We have tried previously and it doesn't work - not applicable specifically for multi-session

How do you deploy it to your AVD (What app type)? Filters/Groups?

**Reply by Champ895** [reply]
> > Cool - and a bit off topic, but do you know how we can get this app specifically working to update within Azure Virtual Desktop? We have tried previously and it doesn't work - not applicable specifically for multi-session
> 
> How do you deploy it to your AVD (What app type)? Filters/Groups?

We tried deploying using the windows app initially through Intune and the configuration profile using admx, but it stated not applicable. We had tried manually pushing it out for winget but it would prompt for admin creds to standard users for installing updates which is what we want to avoid

**Reply by Champ895** [reply]
I just confirmed the check-in status when it's trying to apply it to the VM shows Not applicable. It's being applied to the device. The windows store app installs fine so I think it's related to what admx/adml can be applied to AVD. Not sure if there is a work-around here or if a template that would work in AVD can be developed but it would be very helpful to resolve.

**Reply by Champ895** [reply]
https://learn.microsoft.com/en-us/mem/intune/fundamentals/azure-virtual-desktop-multi-session:

To configure configuration policies for Windows 10 or Windows 11 Enterprise multi-session VMs, you'll need to use the [Settings catalog](https://learn.microsoft.com/en-us/mem/intune/configuration/settings-catalog) in the Microsoft Intune admin center.

**Reply by fedthans** [reply]
any news regarding control over exclude list?
